### PR TITLE
Update build-publish-chainlink.yml

### LINKED ISSUE #3227: [NODE] Dockerfile for arm
Hello, is there docker image for chainlink node that can run on arm cpu ( like Raspberry Pi's)
Thanks,
Love this project,writing my master thesis about chainlink

### PR DESCRIPTION
**Pull request by willianpaixao**
Fix #3227

Adding Docker Buildx plugin enables multi-architecture images, rather than
than only x86_64.

### CODE REVIEW

**Comment by se3000** [comment]
I reopened this as another PR to get around some of the CI permissions issues. There was a request for changes there: https://github.com/smartcontractkit/chainlink/pull/3650#pullrequestreview-533747584

If you can update this branch I'll push it to my PR

**Comment by se3000** [comment]
Also, thanks for the PRs! Getting them worked into the review cycle now.

**Comment by willianpaixao** [comment]
Thanks a lot, @se3000!
What do you want me to update?

**Comment by willianpaixao** [comment]
Hey @se3000 @PatrickAlphaC,

I have made the requested changes and now the Github workflow is able to push to build and push to multiple architectures and still following the current tagging pattern `[branch].[commit hash]` (it was harder than I expcted).

Although, because I wanted to "locally" test, I've named the docker image under my username.
Follow a screenshot of the [temporary repo](https://hub.docker.com/repository/docker/willianpaixao/chainlink).
Screenshot_20201124_004811

Rest assure that I will replace to the original name once all requested changes are done, review and approved.
For now, please review this PR.

**Comment by PatrickAlphaC** [comment]
Thanks for this PR!

File: .github/workflows/main.yml
Comment (HenryNguyen5) [inline .github/workflows/main.yml:39]: This should be set to false for now, we're not currently using GHA to push anything, just keeping it around for when we migrate over

File: .github/workflows/main.yml
Comment (HenryNguyen5) [inline .github/workflows/main.yml:34]: This tag should be changed over to `smartcontractkit`

File: .github/workflows/main.yml
Comment (HenryNguyen5) [inline .github/workflows/main.yml:1]: These changes should be applied to the existing `build-publish-chainlink.yml` file

File: .github/workflows/main.yml
Comment (HenryNguyen5) [inline .github/workflows/main.yml:19]: We use ECR now, please see `build-publish-chainlink.yml`

**Review by HenryNguyen5 — CHANGES_REQUESTED** [review]
Thanks for

File: .github/workflows/main.yml
Comment (willianpaixao) [reply .github/workflows/main.yml:39]: Hey @HenryNguyen5, thanks for the review. As I commented above, some values were temporary so I could push and test the build. https://github.com/smartcontractkit/chainlink/pull/3614#issuecomment-732492420.
Since the overall changes are approved, I'll update the branch.

@HenryNguyen5 will you be able to merge this PR or another one will be created?

**Comment by willianpaixao** [comment]
Forgot that there was a base image and that was also needed to be built in multi-arch, for the final image to work properly. https://github.com/smartcontractkit/builder/pull/61

File: .github/workflows/main.yml
Comment (HenryNguyen5) [reply .github/workflows/main.yml:39]: Yeah thanks for testing it! Please update the branch. When it's ready I will have to fork it and open it again

Comment (willianpaixao) [reply .github/workflows/main.yml:39]: @HenryNguyen5 I think I will wait until the pipeline it's fixed so we can merge the original PRs.

**Comment by samsondav** [comment]
@HenryNguyen5 whats the story here

**Comment by HenryNguyen5** [comment]
@samsondav this PR is dependent on https://github.com/smartcontractkit/builder/pull/61 which never got changes addressed

**Comment by willianpaixao** [comment]
Hey @j16r I saw you closed the PR. All changes were made and that includes https://github.com/smartcontractkit/builder/pull/61 but never got reviewed. Why was it closed?

**Comment by j16r** [comment]
It looks like it had stalled out and couldn't be merged due to https://github.com/smartcontractkit/builder/pull/61 which looks abandoned

**Pull request closed without merging**
